src/bin/jcli_app/mod.rs

```diff
@@ -21,6 +22,8 @@ pub enum JCli {
     Rest(rest::Rest),
     /// Build and view offline transaction
     Transaction(transaction::Transaction),
+    /// Debug tools for developers
+    Debug(debug::Debug),
```

Comment:
It seems to me the `message` command should be merged with the transaction CLI.

`jcli message new ...`
`jcli message decode < message.bin`

Reply:
I was considering this, but then noticed that the output of this command is unstable, developer-only-readable, inconsistent with the rest of CLI (not YAML) and probably not really usable to the end user. That's why I decided to follow jormungandr-cli and put it in debug subcommand.